SupportYourLocals.Data/XMLDataMarketplaces.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using System.Xml.Linq;
+using System.IO;
+using MapControl;
+
+namespace SupportYourLocals.Data
+{
+    public class XMLDataMarketplaces : IMarketStorage
+    {
+        private const string filePath = @"./MarketPlacesData.xml";
+        readonly Dictionary<string, MarketplaceData> dictionaryMarketplaceDataById;
+
+        public XMLDataMarketplaces()
+        {
+            dictionaryMarketplaceDataById = LoadData();
+        }
+
+        public XMLDataMarketplaces(Dictionary<string, MarketplaceData> dictionaryMarketplaceDataById)
+        {
+            this.dictionaryMarketplaceDataById = dictionaryMarketplaceDataById;
+        }
```

Comment:
You should either remove this constructor or add a similar constructor to the other XMLData class.